aio/content/marketing/contributors.json

```
@@ -576,6 +576,13 @@
     "picture": "cexbrayat.jpg",
     "bio": "Author of `Become a ninja with Angular (2+)` https://books.ninja-squad.com/angular - Angular trainer and @Ninja-Squad co-founder"
   },
+  "ajitsinghkaler0": {
```

Please use your GitHub username as key:

```suggestion
  "ajitsinghkaler": {
```